how-to-guides/neptune-docker/scripts/training.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+import neptune.new as neptune
+
+# Step 1: Initialize Neptune and create new Neptune Run
```

Comment:
Here you have `Step n:` which you do not use in other sections. Pls decide on single convention.